books/RayTracingTheNextWeek.html

```diff
@@ -1024,7 +1037,7 @@
         if (!a->bounding_box(0,0, box_a) || !b->bounding_box(0,0, box_b))
             std::cerr << "No bounding box in bvh_node constructor.\n";
 
-        return box_a.min().e[axis] < box_b.min().e[axis];
+        return box_a.axis(axis).min < box_b.axis(axis).min;
```

Comment:
maybe make it more explicit that `axis()` is the function that returns an interval, and `axis` is an int tells that function what to return.
Maybe change the `axis` variable to `a` or `axis_int`

Reply:
Yeah, I can do better than what I've got.